pkgs/by-name/kt/ktx-tools/package.nix

```
@@ -0,0 +1,68 @@
+{
+  cmake,
+  doxygen,
+  fetchFromGitHub,
+  getopt,
+  ninja,
+  lib,
+  pkg-config,
+  stdenv,
+}:
+stdenv.mkDerivation rec {
+  pname = "ktx-tools";
+  version = "4.2.1";
+
+  src = fetchFromGitHub {
+    owner = "KhronosGroup";
+    repo = "KTX-Software";
+    rev = "v${version}";
+    sha256 = "sha256-P44N2ff+3xBwQ3waAIY2c6uANZC4hSdQdSAzyNaBGbs=";
```

```suggestion
    hash = "sha256-P44N2ff+3xBwQ3waAIY2c6uANZC4hSdQdSAzyNaBGbs=";
```
The string already describes the algorithm